pkg/hash/hash.go

```diff
@@ -24,17 +26,17 @@ type (
 
 // Hash160b returns 160-bit (20-byte) hash of input
 func Hash160b(input []byte) Hash160 {
-	// use Blake2b algorithm
-	digest := blake2b.Sum256(input)
+	// use sha3 algorithm
+	digest := crypto.Keccak256(input)
 	var hash Hash160
 	copy(hash[:], digest[7:27])
```

Comment:
i think Ethereum use the latest 20-bytes of hash, that is digest[12:] as the address